### ISSUE TITLE
[mobile] Large header don't display body

### ISSUE THREAD

**Issue by dominiquefournier** [the-djmaze/snappymail#1284]
In phone, when I display a mail where there is a very large header, like a big list of mail recipients, the scroll allow to see all the header, but the body is not displayed and not scrollable.

**Describe the solution you'd like**
It may be interesting to put the header and the body in stack to scroll only one time instead of include body in another div

I use Fenec on Android, I can not reproduce this problem on Firefox on Desktop

**Comment by peterkeist** [comment]
By clicking on the "i" top left in the message you can minimize the header. Does this help?

**Comment by dominiquefournier** [comment]
Yes, you are right (I never click on this, thanks to inform that exists) but is it possible just to need to scroll and display the mail after the header ?
